.github/workflows/ci.yaml

```diff
@@ -571,6 +573,12 @@ jobs:
         with:
           cache-key: ${{ steps.download-cache.outputs.cache-key }}
 
+      - name: Upload Embedded Postgres Cache
+        uses: ./.github/actions/embedded-pg-cache
+        with:
+          cache-key: ${{ steps.download-cache.outputs.cache-key }}
+          cache-path: ${{ runner.os == 'Windows' ? 'C:/temp/embedded-pg-cache' : '/tmp/embedded-pg-cache' }}
```

Comment:
whoops, apparently this isn't valid GitHub

Reply:
I recommend making the cache-path an output of a previous step